.secrets.baseline

```diff
@@ -1,10 +1,10 @@
 {
   "custom_plugin_paths": [],
   "exclude": {
-    "files": "^(.secrets.baseline|.env.example|.github/workflows/ci.yml|README.md|config/brakeman.ignore|config/i18n-tasks.yml|config/locales/.*.yml)$|^.secrets.baseline$",
+    "files": "^(.secrets.baseline|.env.example|.github/workflows/ci.yml|README.md|config/brakeman.ignore|config/i18n-tasks.yml|config/locales/.*.yml)$|^.secrets.baseline$|^package.json$|^package-lock.json",
```

Comment:
urgh secrets... this config fileis sooo ugly...
I wonder if we should recap if this hook is still bringing us value or not?

Reply:
I'm starting to think it isn't.   This file can get especially gnarly when you're rebasing a branch that makes multiple changes to this file.

Reply:
I'll put a ticket in drafts...

Reply:
I'm also very skeptical that this is bringing us value - all our secrets are read from environment variables, and differ per-environment, so it's hard to imagine how we could accidentally commit a real one.